kernel/vmvalues.ml

```diff
@@ -333,6 +336,7 @@ struct
   | RelKey i -> combinesmall 3 (Int.hash i)
   | EvarKey evk -> combinesmall 4 (Evar.hash evk)
 end
+[@@@ocaml.warning "+41"]
```

Comment:
Just using `[@@ocaml.warning "-41"]` here to disable the warning locally (for this structure item `module IdKeyHash = ...`) should work, I believe, or you could disable it within the submodule with `struct [@@@ ...]` (no need for a close then).

Reply:
(I think it's always a good idea to respect warning 41 and fix the code.)